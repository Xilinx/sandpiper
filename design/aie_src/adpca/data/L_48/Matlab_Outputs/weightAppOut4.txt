-6.710373878e+00 -4.057787418e+00
-9.211019516e+00 -6.295640945e+00
-6.926174164e-01 -3.067414761e-01
