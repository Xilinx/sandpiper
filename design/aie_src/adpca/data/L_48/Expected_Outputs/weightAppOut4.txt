-6.710191727e+00 -4.057826042e+00 
-9.210521698e+00 -6.295551777e+00 
-6.926424503e-01 -3.067524433e-01 
